.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "rules": {
+        "indent": [
```

Comment:
can you make this the default 4 space indent (more consistent with other OSS JS repos from our org)?